boards/silabs/dev_kits/xg24_dk2601b/board.c

```diff
@@ -29,15 +29,13 @@ static int efr32xg24_dk2601b_init(void)
 
 
 	if (!gpio_is_ready_dt(&wake_up_gpio_dev)) {
-		LOG_ERR("Wake-up GPIO device was not found!\n");
-		return -ENODEV;
+		LOG_ERR("Wake-up GPIO device was not found!");
 	}
 	ret = gpio_pin_configure_dt(&wake_up_gpio_dev, GPIO_OUTPUT_ACTIVE);
 	if (ret < 0) {
-		return ret;
+		LOG_ERR("Failed to configure wake-up GPIO device!");
 	}
```

Comment:
Just curiosity, what is the purpose of configuration the GPIO in board_late_init_hook() rather than using `output-high` in the device tree?